### PR TITLE
DCS-374 Fixing reminder polling

### PR DESCRIPTION
**Pull request by andrewrlee**
Always set next reminder date if reminder exists. Also adding more logging

### CODE REVIEW

File: job/reminders/reminderPoller.js [hunk 1]
```diff
@@ -1,20 +1,28 @@
 /* eslint-disable no-await-in-loop */
-module.exports = (db, incidentClient, sendReminder, eventPublisher, emailResolver) => {
+const logger = require('../../log')
+
+module.exports = (db, incidentClient, reminderSender, eventPublisher, emailResolver) => {
   const processReminder = async client => {
     const reminder = await incidentClient.getNextNotificationReminder(client)
-
+    logger.info('Found reminder reminder', reminder)
```
Comment (brightonsbox) [inline job/reminders/reminderPoller.js]: Is this meant to say `reminder reminder`? 😄

Comment (andrewrlee) [reply job/reminders/reminderPoller.js]: no - fixed this and refactored this method a bit

File: job/reminders/reminderSender.js [hunk 2]
```diff
@@ -34,6 +34,14 @@ module.exports = (notificationService, incidentClient) => {
     await incidentClient.setNextReminderDate(reminder.statementId, null, client)
   }
 
+  const setNextReminderDate = async (client, reminder) => {
```
Comment (brightonsbox) [inline job/reminders/reminderSender.js]: Does this need a corresponding test?

Comment (andrewrlee) [reply job/reminders/reminderSender.js]: It's already covered - it's just extracting out a method of code that already existed

**Pull request merged**